release-github/action.yml

```
@@ -347,23 +347,22 @@ runs:
       env:
         LIBRARY_NAME: ${{ inputs.library-name }}
       run: |
-        SDIST_FILE=$(find dist/${LIBRARY_NAME}-artifacts -name '*.tar.gz' | sort | tail -n 1 | xargs basename)
-        WHEEL_FILE=$(find dist/${LIBRARY_NAME}-artifacts -name '*.whl' | sort | tail -n 1 | xargs basename)
 
         {
           echo "RELEASE_NOTES_BODY<<EOF"
           echo "${RELEASE_NOTES_BODY}"
-
+  
           echo "### Artifact Attestations"
           echo ""
-          echo "You can verify this release's artifact attestations using [GitHub's command-line tool](https://cli.github.com/). For example, download the source distribution and wheel from this release page or from [PyPI](https://pypi.org/). Then, run the following commands:"
+          echo "You can verify this release's artifact attestations using [GitHub's command-line tool](https://cli.github.com/). For example, download the files from this release page or from [PyPI](https://pypi.org/). Then, run the following commands:"
           echo ""
           echo "\`\`\`bash"
-          echo "$ gh attestation verify --owner ${{ github.repository_owner }} ${SDIST_FILE}"
-          echo "\`\`\`"
-          echo "and"
-          echo "\`\`\`bash"
-          echo "$ gh attestation verify --owner ${{ github.repository_owner }} ${WHEEL_FILE}"
+  
+          # Find the following artifacts: wheels, source distribution, and wheelhouse files
+          for artifact in $(find dist -type f \( -name "*.whl" -o -name "*.tar.gz" -o name "*.zip" \)); do
```

```suggestion
          for artifact in $(find dist -type f \( -name "*.whl" -o -name "*.tar.gz" -o name "*wheelhouse*.zip" \)); do
```
I'm fearing that documentation artifacts are uploaded as ZIP files

---

Ah good catch but... aren't they signed too? We can also provide signed docs 😆

---

Yeah, I understood after typing and edited but you answered faster :')